docs/source/project_documentation/user_guide/model_definitions.md

```diff
@@ -35,9 +35,9 @@ To compose a system from individual `component`s, one specifies additional
 
 ### Defining System Behavior In Models
 
-Each model should define one, or more, `behavior`s that describe the component's
-behaviors. These behaviors include a descrpition of `input`s, `output`s, and
-`scenario`s that show how the model interacts with different components in the
+Each `model` should define one, or more, `behavior`s that describe the component's
+behaviors. These behaviors include a description of `input`s, `output`s, and
+`scenario`s that show how the `model` interacts with different components in the
```

Comment:
Similar to below, this isn't referencing the `model` field, but model as a whole. 

```suggestion
Each model should define one, or more, `behavior`s that describe the component's
behaviors. These behaviors include a description of `input`s, `output`s, and
`scenario`s that show how the model interacts with different components in the
```